src/database/DataTypes/Name.js

```diff
@@ -56,7 +56,7 @@ export class Name extends Realm.Object {
     const sumOfCredits = this.customerCredits.sum('outstanding');
     const sumOfDebits = this.customerInvoices.sum('outstanding');
 
-    return Math.abs(sumOfDebits - sumOfCredits);
+    return Math.abs(sumOfCredits + sumOfDebits);
```

Comment:
hm? I'm not following the logic. I would have thought that the available credit is `sumOfCredits - sumOfDebits` though I'm not convinced that mSupply is actually doing that!

Reply:
ah yes. that makes sense, thanks